content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -275,6 +275,16 @@ Multus supports all [reference plugins](https://github.com/containernetworking/p
 
 [OVN4NFV-K8S-Plugin](https://github.com/opnfv/ovn4nfv-k8s-plugin) is OVN based CNI controller plugin to provide cloud native based Service function chaining(SFC), Multiple OVN overlay networking, dynamic subnet creation, dynamic creation of virtual networks, VLAN Provider network, Direct provider network and pluggable with other Multi-network plugins, ideal for edge based cloud native workloads in Multi-cluster networking
 
+### Netris
+
+The [Netris software controller](https://www.netris.ai/kubernetes/) allows NetOps/DevOps engineers to design and operate a physical network with IaC (Infrastructure as Code), a simple web interface, or by using a modern REST API. Netris has native integration for Kubernetes with which provides automatic creation of network services using CRD based objects:
+
+* Automatically creating L4LB resources for [services of type LoadBalancer](https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/)
```

Comment:
L4LB is a not a term widely used in the k8s community.
Use relative path (i.e. `/docs/tasks/...`) rather than full path.